client/src/app/+videos/+video-edit/video-add-components/video-import-url.component.html

```diff
@@ -20,19 +20,30 @@
 
     <div class="form-group">
       <label i18n for="first-step-channel">Channel</label>
-      <div class="peertube-select-container">
-        <select id="first-step-channel" [(ngModel)]="firstStepChannelId" class="form-control">
-          <option *ngFor="let channel of userVideoChannels" [value]="channel.id">{{ channel.label }}</option>
-        </select>
+      <div class="peertube-ng-select-container">
+        <ng-select
+          labelForId="first-step-channel" [(ngModel)]="firstStepChannelId"
+          bindLabel="label" bindValue="id" [clearable]="false" [searchable]="false"
+        >
+          <ng-option *ngFor="let channel of userVideoChannels" [value]="channel.id">
+            <img class="mr-1" [src]="channel.avatar" height="20px" width="20px"/>
+            {{ channel.label }}
+          </ng-option>
+        </ng-select>
       </div>
     </div>
 
     <div class="form-group">
       <label i18n for="first-step-privacy">Privacy</label>
-      <div class="peertube-select-container">
-        <select id="first-step-privacy" [(ngModel)]="firstStepPrivacyId" class="form-control">
-          <option *ngFor="let privacy of videoPrivacies" [value]="privacy.id">{{ privacy.label }}</option>
-        </select>
+      <div class="peertube-ng-select-container">
+        <ng-select
```

Comment:
Could you create a dedicated component for a select + description? (so we just have to send an object)